test/server/cards/02_SHD/upgrades/InfiltratorsSkill.spec.js

```diff
@@ -0,0 +1,22 @@
+describe('Infiltrator\'s Skill', function() {
+    integration(function() {
+        describe('Infiltrator\'s Skill\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'battlefield-marine', upgrades: ['infiltrators-skill'] }],
+                    },
+                    player2: {
+                        groundArena: ['pyke-sentinel']
+                    }
+                });
+            });
+
+            it('should ignore sentinel', function () {
+                this.player1.clickCard(this.battlefieldMarine);
+                expect(this.player1).toBeAbleToSelectExactly([this.pykeSentinel, this.p2Base]);
```

Comment:
Can we confirm that it can actually resolve the attack against base (do the click and confirm that the damage happens)